default.nix

```diff
@@ -91,6 +91,12 @@ rec {
     inherit haskell webCommon webCommonPlutus;
   });
 
+  # Cardano
+  inherit (plutus.haskell.project.hsPkgs.cardano-cli.components.exes) cardano-cli;
+  cardano-node = plutus.haskell.project.hsPkgs.cardano-node.components.exes.cardano-node // {
+    deployments = pkgs.cardano-node-deployments;
```

Comment:
This definitely needs a comment!

Reply:
This incantation was lifted from the `cardano-wallet` project, I don't know what it does yet :)